app/views/provider/admin/dashboards/_service.html.slim

```diff
@@ -1,9 +1,10 @@
-section.DashboardSection.DashboardSection--service class=('DashboardSection--wide' unless can?(:manage, :partners)) class=('DashboardSection--toggleable is-closed u-legacy-cookie' if current_account.multiservice?) id=(dom_id service)
+section.DashboardSection.DashboardSection--service class=('DashboardSection--wide' unless can?(:manage, :analytics)) class=('DashboardSection--toggleable is-closed u-legacy-cookie' if current_account.multiservice? && can?(:manage, :analytics)) id=(dom_id service)
```

Comment:
The idea is good 👍 I don't know about removing partners from there but I believe you 😄 
About coding, a minor thing is I wouldn't call to `can?(:manage, :analytics)` and `current_account.multiservice? && can?(:manage, :analytics))` so many times in the same file. I think it is better something like this at the beginning of the file and later use this vars:
```slim
- can_manage_analytics = can?(:manage, :analytics)
- can_manage_analytics_with_multiservice = current_account.multiservice? && can_manage_analytics
```